extensions/uuid-generator/package.json

```diff
@@ -7,7 +7,8 @@
   "author": "jmaeso",
   "contributors": [
     "GastroGeek",
-    "pernielsentikaer"
+    "pernielsentikaer",
+    "LunaticMuch"
```

Comment:
this is not the right file in the project.

Reply:
Correct 😊